rust/utils/run-locally/src/cosmos/crypto.rs

```diff
@@ -1,3 +1,7 @@
+// TODO: this file can be removed if `CosmosAddress` can be imported from `hyperlane-cosmos`.
```

Comment:
I assume this is what you're currently looking into?

Reply:
actually no but I'll look into this as well